integrations/api_activitypub_person_test.go

```diff
@@ -75,10 +75,10 @@ func TestActivityPubMissingPerson(t *testing.T) {
 
 func TestActivityPubPersonInbox(t *testing.T) {
 	setting.Federation.Enabled = true
-	c = routers.NormalRoutes()
+	c = routers.NormalRoutes(context.Background())
```

Comment:
Why background here? *It's inconsistent*

Reply:
It's a test.